styles/variables-silicon.less

```diff
@@ -116,6 +116,7 @@
 @agate-dropdown-button-height: 60px;
 @agate-dropdown-button-padding: 0 15px;
 @agate-dropdown-item-border-width: 0;
+@agate-dropdown-item-height: 42px;
```

Comment:
I think this should be 60px

Reply:
I wanted to keep the same height as it is now on develop (42px), but with the new specifications it is 60px. I changed it.